src/components/icons/circle-prod.tsx

```diff
@@ -0,0 +1,16 @@
+export const CircleProdIcon = () => (
+	<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" fill="none">
+		<rect width="16" height="16" rx="8" fill="#4AB866" fillOpacity="0.4" />
```

Comment:
Adding a `title` is a good a11y practice.
```suggestion
        <title>Production Environment</title>
		<rect width="16" height="16" rx="8" fill="#4AB866" fillOpacity="0.4" />
```

Reply:
Mentioned it in this issue, since necessary to test it carefully and also address localization
STU-561